<launch>

  <!-- Bringup minimal -->

  <include file="$(find turtlebot_bringup)/launch/minimal.launch">
  </include>
	
	<!-- set the odometry correction parameter! -->
  <param name="/turtlebot_node/odom_angular_scale_correction" value="1.006" />


	<!-- Amcl navigation -->
	
	<include file="$(find rins_navigation)/launch/amcl_demo.launch">
	</include>
	
	
	<!-- Cylinder segmentation -->
	<node name="cylseg" pkg="task3" type="cylinder_segmentation" respawn="false"/>
	<node name="call_cylseg" pkg="task3" type="call_cylinder_detection.py" respawn="false"/>
	
	<!-- Ring detection -->
	<node name="ring_detection" pkg="task3" type="detect_rings_reformated.py" respawn="false"/>
	
	<!-- QR codes -->
	<node name="qr_detection" pkg="task3" type="check_for_qr.py" respawn="false"/>
	
	<!-- check if reachable service -->
	<node name="check_if_reachable" pkg="task3" type="check_if_reachable.py" respawn="false"/>
	
	<!-- check for numbers -->
	<node name="check_for_numbers" pkg="task3" type="check_for_numbers.py" respawn="false"/>
	
	<!-- accumulate positions -->
	<node name="accumulate_positions" pkg="task3" type="accumulate_positions.py" respawn="false"/>
	
	<!-- check color -->
	<node name="check_color" pkg="task3" type="barva_cilindra.py" respawn="false"/>
	
	
</launch>
